### ISSUE TITLE
Failed to fetch..

### ISSUE THREAD

**Issue by skyport** [chaiNNer-org/chaiNNer#1069]
Hi Joey,
Sorry, unfortunately not fixed and now there is another error than the first time. When running the program for the first time, it tries to download dependencies and this error occurs.
Bests,

<img width="1278" alt="Failed to Fetch" src="https://user-images.githubusercontent.com/6561012/194763768-73bd4f20-7101-482a-9ca7-e3420db1aafd.png">

**Comment by RunDevelopment** [comment]
@skyport Could you please attach Chainner's logs? In the menu, go *Help > Open logs folder* and you'll see a 2 log files.

**Comment by skyport** [comment]
Sure.. Thanks. 

[renderer.log](https://github.com/chaiNNer-org/chaiNNer/files/9742158/renderer.log)
[main.log](https://github.com/chaiNNer-org/chaiNNer/files/9742159/main.log)
<img width="1436" alt="waiting like this for half an hour" src="https://user-images.githubusercontent.com/6561012/194770101-35051e22-d445-4a43-9ec9-3bbfa2569e72.png">

**Comment by joeyballentine** [comment]
ONNX cannot be installed on M1 devices. I'm confused though, you get the failed to fetch problem but are still able to access the application?

**Comment by skyport** [comment]
yes exactly. I can even upscale as standard. But without running any model. I can't run it even if I wanted to. If you want, I can shoot a video for you and send it in a spare time. 

In fact, if you examine the log carefully, you can see the whole situation, as you understand.

There is something new coming out every day. I greatly appreciate your effort. For this reason, I want it to work completely as soon as possible.
My question is this. When will this situation be fully resolved? Because due to a few problems, I think that a program with great potential that everyone can use cannot find the place it deserves.
Best

**Comment by joeyballentine** [comment]
Looking again at the logs, it appears you are missing a library that torchvision is trying to load, called libpng. 

I think this will sorta be fixed in the next update since any nodes that use modules that fail to import will simply just be ignored and the rest loaded properly. The root issue though is that dependencies we rely on are incompatible with some systems. If you have a way of installing libpng, like through homebrew, you could try that.

**Comment by AugmentedRealityCat** [comment]
I have the same problem but the app closes before I can do anything. Since the app won't open after the 'failed to fetch' error that is identical to the screenshot at the top of this thread, I cannot open the HELP menu to access log files.

I found two log files, one for links creation, and the other for app update, but there seems to be nothing in there.

Should I try an earlier version ?

**Comment by AugmentedRealityCat** [comment]
Finally, even though the app did start, I was not able to install all the missing dependencies all at once. The progress bar for installing them from the dependency manager would almost fill, then when at the very end it would go back to the beginning and do that a few times, but there would still be missing things. Each time I would restart it some packages would get installed , and after doing that quite a few times I got everything to install. 

Patience and repetition was the solution for me !

**Comment by joeyballentine** [comment]
Interesting... @AugmentedRealityCat what operating system are you on? This sounds like some sort of weird pip-related issue. Glad it finally worked, but I definitely need to figure out what's causing that behavior in the first place

**Comment by AugmentedRealityCat** [comment]
I'm on Windows 10.

**Comment by joeyballentine** [comment]
Honesty I have no clue why it would be acting so strangely for you then. If I had to guess, it's just something weird about your system.

The progress bar almost filling up and then going back down is normal, that's because it resets each time it installs one of the sub-dependencies. I wanted to make it all one progress bar but I just haven't gotten around to it yet.

**Comment by leonwormuth** [comment]
Hi, I am also on Windows 10 and facing a similar issue so I thought I'd add my logs here in case they can help at all.
[main.log](https://github.com/chaiNNer-org/chaiNNer/files/9828759/main.log)
[renderer.log](https://github.com/chaiNNer-org/chaiNNer/files/9828760/renderer.log)

This only started happening after I installed PyTorch via the chaiNNer UI.

I get the startup splash with loading text, followed by an error alert saying "Failed to fetch" as shown below. I can't get past the error alert at all, closing it via the x button also closes the application.
![image](https://user-images.githubusercontent.com/20897695/196921648-5d5d7806-dbf6-49e3-9d9b-1758ce18eb5e.png)
![image](https://user-images.githubusercontent.com/20897695/196921916-10b7b8d5-e4f7-41dc-9fb6-265d7dc46f9a.png)

**Comment by joeyballentine** [comment]
@leonwormuth can you download 0.13.0 and see if the same issue happens on that version as well?

**Comment by leonwormuth** [comment]
> @leonwormuth can you download 0.13.0 and see if the same issue happens on that version as well?

On 0.13.0 I don't get that error upon restart.

However I get the below, which seems to be related to issue #831 (I get the same error in my log).

![image](https://user-images.githubusercontent.com/20897695/197012488-50f3e631-e659-4612-8884-9e968278bfc4.png)
